src/herder/test/TransactionQueueTests.cpp

```diff
@@ -716,7 +716,7 @@ TEST_CASE("transaction queue starting sequence boundary",
         for (size_t i = 1; i <= size; ++i)
         {
             REQUIRE(txSet->mTransactions[i - 1]->getSeqNum() ==
-                    startingSeq - 3 + i);
+                    static_cast<int64_t>(startingSeq - 3 + i));
```

Comment:
maybe the right fix here is to use `int` for `i` instead of `size_t`

Reply:
bleh, this ends up being equally bad (because of the `REQUIRE`):
```c++
auto checkTxSet = [&](uint32_t ledgerSeq, int64_t size) {
        auto lcl = app->getLedgerManager().getLastClosedLedgerHeader();
        lcl.header.ledgerSeq = ledgerSeq;
        auto txSet = tq.toTxSet(lcl);
        REQUIRE(txSet->mTransactions.size() == static_cast<size_t>(size));
        for (int64_t i = 1; i <= size; ++i)
        {
            REQUIRE(txSet->mTransactions[i - 1]->getSeqNum() ==
                    startingSeq - 3 + i);
        }
    };
```

Reply:
let's keep your version then